Kubernetes/omsagent.yaml

```diff
@@ -130,8 +130,6 @@ spec:
        volumeMounts:
         - mountPath: /var/run/docker.sock
           name: docker-sock
-        - mountPath: /var/log 
-          name: host-log
```

Comment:
Please hold off on submitting these changes. With the recent changes to fluent-bit running in omsagent container itself, we need this to be mounted.